packages/node/test/checkin.test.ts

```diff
@@ -44,7 +44,7 @@ describe('CheckIn', () => {
         duration: 10.0,
         release: '1.0.0',
         environment: 'production',
-      } as CheckIn,
+      } as SerializedCheckIn,
```

Comment:
giga l: you can simply do `as const` instead of `as SerializedCheckIn` here. TS is only complaining because `status` is interpreted as `string`.

Reply:
Gonna keep this anyway because I prefer the explicitness of the type.